tests/components/find_replace_side_panel.test.ts

```diff
@@ -146,7 +146,6 @@ describe("find and replace sidePanel component", () => {
     test("search match count is removed when input is cleared", async () => {
       setCellContent(model, "A1", "Hello");
       setInputValueAndTrigger(selectors.inputSearch, "Hel", "input");
-      await nextTick();
```

Comment:
the commit has changed the bahaviour of the input. we directly call the function that updates the search instead of having to wait for a timer. So at the following rendering,( that is nextTick() )  the component is actually updated.
It makes sense to test the new behaviour rather than removing a line of code to have the old test pass, don't you think?